internal/hud/webview/view.go

```diff
@@ -118,17 +123,27 @@ func ToWebViewBuildRecord(br model.BuildRecord) BuildRecord {
 	}
 }
 
+func timeToProto(t time.Time) *timestamp.Timestamp {
+	ts, err := ptypes.TimestampProto(t)
+	if err != nil {
+		return nil
```

Comment:
this seems dangerous. i would be more comfortable with this if it was a `panic()`, tho it really should return an error

Reply:
@nicks the proto know show to handle `nil` as the empty value for a timestamp, so I think this is OK.

https://github.com/golang/protobuf/blob/master/ptypes/timestamp.go#L95

Reply:
oh, my concern isn't that we're returning nil, but that we're quietly dropping validation errors

Reply:
And in fact empty values is what the web UI expects here.

Reply:
Ah, okay. I convert all of these things to return error then.